### ISSUE TITLE
Cannot set properties of undefined (setting 'workerSrc')

### ISSUE THREAD

**Issue by panwaranuj01** [wojtekmaj/react-pdf#1662]
### Before you start - checklist

- [X] I followed instructions in documentation written for my React-PDF version
- [X] I have checked if this bug is not already reported
- [X] I have checked if an issue is not listed in [Known issues](https://github.com/wojtekmaj/react-pdf/wiki/Known-issues)
- [X] If I have a problem with PDF rendering, I checked if my PDF renders properly in [PDF.js demo](https://mozilla.github.io/pdf.js/web/viewer.html)

### Description

TypeError: Cannot set properties of undefined (setting 'workerSrc')\n    at ./node_modules/react-pdf/dist/esm/index.js 

```
import React, { useState } from "react";
import { Document, Page, pdfjs } from 'react-pdf';

import "react-pdf/dist/esm/Page/AnnotationLayer.css";
import "react-pdf/dist/esm/Page/TextLayer.css";
pdfjs.GlobalWorkerOptions.workerSrc = `//unpkg.com/pdfjs-dist@${pdfjs.version}/legacy/build/pdf.worker.min.js`;

```

### Steps to reproduce

I have just imported this 

### Expected behavior
My react app should load

### Actual behavior
My react app is not loading

### Environment

- **Browser (if applicable)**:
- **React-PDF version**: "^7.5.1"
- **React version**:" 16.9.0"
- **Webpack version (if applicable)**:

**Comment by tonykhaov** [comment]
Hey did you check that you don't have `pdfjs-dist` in your package.json? I had the same issue and fixed it by removing pdfjs-dist dependency

**Comment by wojtekmaj** [comment]
Duplicate of #1579

**Comment by neerajkatiyar** [comment]
@wojtekmaj Could you please provide us a fix for v5.7.2

**Comment by wojtekmaj** [comment]
> @wojtekmaj Could you please provide us a fix for v5.7.2

Sorry, not a chance. This version is no longer supported. I don't have capacity to maintain multiple versions.

**Comment by neerajkatiyar** [comment]
No problem, I can understand, we will upgrade to the latest version.
Thanks
